.vib/thanos/queryfrontend/cypress/cypress/integration/thanos_queryfrontend_spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+  cy.visit('/');
+  cy.contains('.dropdown-toggle', 'Status').click();
+  cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+  cy.contains('Runtime Information').should('be.visible');
+  cy.get('h2').contains('Build Information').should('be.visible');
+})
+
+it('allows the execution of a query', () => {
+  cy.visit('/graph');
+  cy.get('.alert').contains('No data queried yet').should('be.visible');
+  cy.get('.cm-line').type('scalar');
+  cy.contains('.cm-completionLabel', 'scalar').click();
+  cy.get('.execute-btn').click();
+  cy.get('.alert').contains('Empty query result').should('be.visible');
+})
+
+it('allows listing all installed stores', () => {
+  cy.visit('/');
+  cy.contains('[class="nav-link"]', 'Stores').click();
+  cy.get('[data-testid="endpoint"]').should('be.visible');
+  cy.get('[data-testid="health"]').should('be.visible');
```

Comment:
You should also check as well that the Store is reachable (the status is OK)

Reply:
![Screenshot 2022-04-05 at 15 32 23](https://user-images.githubusercontent.com/47592680/161765375-865ffddd-4447-401a-851d-ad919037eb8a.png)
This is checking for the OK icon, which means the test is already covering that.

Reply:
Oh, now I see! 
Fixed it.